tests/dom_tests.js

```diff
@@ -174,6 +174,11 @@ QUnit.test("matches", function(assert) {
   assert.ok(document.querySelector('#baz').matches('.beta'));
 });
 
+QUnit.test("closest", function(assert) {
+  assert.equal(document.querySelector('#bar').closest('#foo'), document.querySelector('#foo'));
+  assert.equal(document.querySelector('#baz').closest('#foo'), document.querySelector('#foo'));
+});
```

Comment:
Can you add a case for:

`assert.equal(document.querySelector('#foo').closest('#foo'), document.querySelector('#foo'));`

And a case where there is no closest ancestor?

(This will also serve as a reminder to re-review and merge this when I have more time)